styled/auth.js

```diff
@@ -1,33 +1,87 @@
 import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
 
-export const SignUpContainer = styled.View`
+// TODO @tommypoa: Consider creating re-usable containers taking in props for the 3 Auth screens
+
+// StartUpScreen
+
+export const StartUpContainer = styled.View`
   display: flex;
   justify-content: center;
   align-items: center;
   background-color: #fff;
+  margin-top: 241px;
```

Comment:
The original goal with the positioning of this button was to keep it at the same position across each screen, which would rest right above the keyboard. We can look into how to do this later and stick to top margins for now, but just keep that in mind.

Reply:
Added a todo for now!